UnitsNet/CustomCode/UnitClasses/Density.extra.cs

```diff
@@ -20,28 +20,61 @@
 // THE SOFTWARE.
 
 
-#if !WINDOWS_UWP
-// Operator overloads not supported in Universal Windows Platform (WinRT Components)
+
 
 namespace UnitsNet
 {
+#if WINDOWS_UWP
+    public sealed partial class Density
+#else
     public partial struct Density
+#endif
     {
+
+#if WINDOWS_UWP
+        internal
+#else
+        public
+#endif
+            Density(Molarity molarity, Mass molecularWeight)
```

Comment:
I don't think we should add a constructor for this. The convention is for the constructor to set the base unit value, so instead the `FromMolarity()` method below should call the default generated constructor with the kg/m³ value you calculate here, or use the `FromKilogramsPerCubicmeter()` constructor method.